airflow/contrib/hooks/aws_hook.py

```diff
@@ -78,35 +78,36 @@ class AwsHook(BaseHook):
     This class is a thin wrapper around the boto3 python library.
     """
 
-    def __init__(self, aws_conn_id='aws_default'):
+    def __init__(self, aws_conn_id='aws_default', region_name=None):
         self.aws_conn_id = aws_conn_id
+        self.region_name = region_name
```

Comment:
This is obtained from the extra connectin parameters, why have it in the signature?